meson.build

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Endless Mobile, Inc.
+
+project('eos-knowledge-renderer', 'c', version: '0.0.0', license: 'LGPL2.1+',
```

Comment:
Consider renaming this to `eknr` (and `eos-knowledge-renderer.h` to `eknr.h` or something like that); with ekncontent I always found it annoying that there were multiple names for the same library.